VTS fix added additional nodes for usb passthrough

bkothapx opened the pull request:
fixed vts test SuspendSepolicyTests added addtional nodes for ADL-PS device
showing for the test exectuion.

Tracked-On: OAM-111651

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

JeevakaPrabu reviewed (changes requested):
please update the commit message with details of the devices added. Replace ADL-PS with ADL-P and ADL-S

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

bkothapx committed:
VTS fix added additional nodes for usb-passthrough

Added genfs rules for ADL-PS, ADL-P and ADL-S devices
With USB-PASSTRHOUGH showing ACPI LNXSYSTM/LNXSYBUS and USB wake nodes

fix VTS test SuspendSepolicyTests

Tracked-On: OAM-111651
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details
